Synview-backend/middleware/jwt.ts

```diff
@@ -0,0 +1,17 @@
+export default function getToken(auth: string) {
+  const authorization = auth
+  if (!authorization) {
+    return null;
+  }
+
+  const [method, token] = authorization.split(" ");
+
+  if (method !== "Bearer") {
+    return null;
+  }
+  if (!token) {
+    return null;
+  }
+
+  return token;
+}
```

Comment:
It is a big confusing to put this `jwt.ts` file in the `middleware` folder, all it is is a function and it is only ever called from `auth_middleware.ts`, so I would just put that function in there